javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,39 @@
+const notes = [];
+function saveNote(content, id) {
+  let obj = { content, id };
+  notes.push(obj);
+  return;
+}
```

Comment:
no need to create `obj` you can do it like this also
```js
  function saveNote(content, id) {
    notes.push({ content, id });
  }
```
and remove `return;` it has no effect at the end of any function